CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+
+### 23.3.2 [#702](https://github.com/openfisca/openfisca-core/pull/702)
+
+Minor Change without impacts for reusers:
+  - Make code more Python3-like by backporting unicode litterals.
+  - With this backport, all strings are by default unicodes.
+  - The `u` prefix for strings should *not* be used anymore.
```

Comment:
If you can give me a shell command that does it, I can add it to Circle 🙂.

Reply:
I know [black](https://github.com/ambv/black) does it automatically... but the change is a bit more involved... as you can see in https://github.com/openfisca/openfisca-france/pull/1060 (and specifically the commit https://github.com/openfisca/openfisca-france/pull/1060/commits/9dd758080c5efd20322093546852d1f1ad49e787 adding it, and the consequences: https://github.com/openfisca/openfisca-france/pull/1060/commits/6d8621b1d470e92a50de12ac92ff35d99803f228)

Reply:
black seems very opinionated, so using it would be something to think about more thoroughly.

Or we can hope that shell wizzard @MattiSG drops by and finds a solution to this issue and the [other one](https://github.com/openfisca/openfisca-core/pull/702#discussion_r207300821) by the wave of a magic wand...